src/mapblock.h

```diff
@@ -31,6 +31,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "nodetimer.h"
 #include "modifiedstate.h"
 #include "util/numeric.h" // getContainerPos
+#include "settings.h"
```

Comment:
As the function that includes it is inline, I can't seem to be able to move it to the cpp file.